src/OVAL/probes/unix/linux/rpmverifyfile.c

```diff
@@ -76,6 +76,18 @@ struct rpmverify_res {
 #define RPMVERIFY_SKIP_GHOST  0x2000000000000000
 #define RPMVERIFY_RPMATTRMASK 0x00000000ffffffff
 
+#ifndef RPMVERIFY_FILEDIGEST
```

Comment:
I think, you cannot check enum value in the #ifdef, it will be always not defined.